mappings/net/minecraft/client/render/block/BlockModelRenderer.mapping

```diff
@@ -130,9 +171,22 @@ CLASS net/minecraft/unmapped/C_agoaazwy net/minecraft/client/render/block/BlockM
 			ARG 3 direction
 			ARG 4 flip
 	CLASS C_wkjcjesl NeighborData
+		FIELD f_dazptjqi vertex0Weights [Lnet/minecraft/unmapped/C_agoaazwy$C_nzndnvsq;
+		FIELD f_nbccphut vertex2Weights [Lnet/minecraft/unmapped/C_agoaazwy$C_nzndnvsq;
+		FIELD f_ocdiblec vextex1Weights [Lnet/minecraft/unmapped/C_agoaazwy$C_nzndnvsq;
```

Comment:
shouldn't vertex1Weights come before vertex2Weights? This arrangement is odd.

Reply:
it is, but its how the fields are defined. I think hashed might be to blame for this

Reply:
Ah okay, no worries then.